### DISCUSSION TITLE
Node level Regression example

### DISCUSSION THREAD

**Discussion by rajshree8217** [danielegrattarola/spektral#412] [Q&A]
You have a graph level regression example with QM9 data , is it possible to do a node level regression prediction for a node feature ?
Do you have an example that you can share .

Thanks
RR

**Comment by danielegrattarola** [comment]
Sure, the main change you have to do is to the global pooling layer. 

You can either remove it, and do full node predictions, or you can select the node feature you want to regress using indices. 
In the latter case, the selection of the feature depends on whether you are using batch mode or disjoint mode (see here https://graphneural.network/data-modes/). In batch mode it's easy, you just select the i-th feature with `x[..., i]`. 
In disjoint mode it's a bit more tricky, I would have to think about it. Let me know if this is the case you're interested in.
